components/bl-simpleModal-component/styles/index.less

```diff
@@ -0,0 +1,208 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
+@bl-customComponent-simple-modal-themeColor: @themePrimary;
+@bl-customComponent-simple-modal-backgroundColor: @appBackgroundColor;
+@bl-customComponent-simple-modal-textColor: @appTextColor;
+
+@bl-customComponent-simple-modal-max-width: 600px;
+@bl-customComponent-simple-modal-min-width: 300px;
+
+@bl-customComponent-simple-modal-buttonColor: @bl-customComponent-simple-modal-themeColor;
+@bl-customComponent-simple-modal-button-backgroundColor: transparent;
+@bl-customComponent-simple-modal-button-backgroundColorHover: fade(@bl-customComponent-simple-modal-themeColor, 4%);
+@bl-customComponent-simple-modal-button-borderRadius: 4px;
+
+body.active-modal {
+  overflow-y: hidden;
+}
+
+.bl-customComponent-simple-modal {
+  z-index: 1;
```

Comment:
I assume some components might have higher z-index and as a result they can overlay the modal 

however, with using ReactProtal the problem should not appear